source/extensions/filters/listener/proxy_protocol/proxy_protocol.cc

```diff
@@ -124,8 +123,6 @@ void Filter::onReadWorker() {
     socket.setRemoteAddress(proxy_protocol_header_.value().remote_address_);
   }
 
-  // Release the file event so that we do not interfere with the connection read events.
```

Comment:
This comment indicates that another consumer of `ioHandle` would create a file event. If this is the case we need to expose a `reset` method from `ioHandle`

Reply:
Turns out if we dont reset we cause a bunch of places to crash.